mlir/tools/miopen-gen/miopen-gen.cpp

```diff
@@ -1186,6 +1547,7 @@ static void emitPrintTensor(OpBuilder &b, mlir::Value var) {
 static func::FuncOp
 createVerifierFunc(ModuleOp &module, const KernelIF &kernel,
                    const miopen::Conv2dGenerator::Config &genConfig) {
+
```

Comment:
I think you do not need to bother to change this function, since I have replace it with the new one.

Reply:
I didn't change the verifier function. And your latest changes have been merged. Thank you.